platform/kube/inject/testdata/hello-config-map-name.yaml.injected

```diff
@@ -61,6 +61,7 @@ spec:
         name: istio-config
       - emptyDir:
           medium: Memory
+          sizeLimit: "0"
```

Comment:
@ayj any ideas how to deal with this?
I filed a bug https://github.com/kubernetes/client-go/issues/257 but that might take a long time to get fixed.